docs/plugins/form-builder.mdx

```diff
@@ -430,10 +430,26 @@ formBuilderPlugin({
         plural: 'Custom Text Fields',
       },
     },
+    country: {
+      interfaceName: 'CountrySelect',
+    },
   },
 })
 ```
 
+<Banner type="warning">
+  Some field names defined by this plugin may result in GraphQL type name collisions with your own Blocks or Collections.
+
+  Example: `Error: Schema must contain uniquely named types but contains multiple types named "Country"`.
+
+  To resolve this, you can:
+
+  1. Change the `slug` in your [Collection Config](https://payloadcms.com/docs/configuration/collections#config-options)  
+  2. Override `graphQL.singularName` in your [Collection Config](https://payloadcms.com/docs/configuration/collections#graphql)  
+  3. Override `interfaceName` in your [Block Config](https://payloadcms.com/docs/fields/blocks#block-configs)  
+  4. Override `interfaceName` in the plugin's field config, as shown above
```

Comment:
change to:

In the case that any fields from this plugin interfere with other blocks, globals or collections in your config via generated types or schema you can fix this by overriding the interface names:
  
- `interfaceName` in the plugin's field config
- `graphQL.singularName` if it interferes with GraphQL schema